Amq_List_Broker-main/activemq.py

```diff
@@ -0,0 +1,156 @@
+import  boto3
+import requests
+import json
+import os
+from botocore.exceptions import ClientError
+import xml.etree.ElementTree as ET
+
+client_Name = os.environ['client_Name']
+send_to = os.environ['SendToEmail']
+mq_user_Name = os.environ['broker_userName']
+mq_password = os.environ['broker_Password']
+
+
+def lambda_handler(event, context):
+    try:
+        clientName= os.environ['client_Name']
```

Comment:
client_Name is already available instead of fetching again from env variable

Reply:
done